lib/Alchemy/Phrasea/Webhook/Processor/OrderNotificationProcessorFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Alchemy\Phrasea\Webhook\Processor;
+
+use Alchemy\Phrasea\Application;
+
+class OrderNotificationProcessorFactory implements ProcessorFactory
+{
+    /**
+     * @var Application
+     */
+    private $application;
+
+    public function __construct(Application $application)
```

Comment:
You can also inject 2 locators for the repos. This way you avoid injecting full application.
`new LazyLocator($app, 'repo.orders')`. The annoying side of it is that you should call `call_user_func($this->orderRepositoryLocator)` or `$locator = $this->orderRepositoryLocator; $orderRepository = $locator();`